src/ontology/fbbt-odk.yaml

```diff
@@ -34,11 +34,28 @@ components:
     - filename: hemibrain_new_ALLNs.owl
     - filename: Feng_neurons.owl
 import_group:
+  use_base_merging: TRUE
+  exclude_iri_patterns:
+    - <http://purl.obolibrary.org/obo/UBERON_*>
   products:
     - id: go
+      make_base: TRUE
+      base_iris:
+        - http://purl.obolibrary.org/obo/GOCHE_
+        - http://purl.obolibrary.org/obo/GO_
+        - http://purl.obolibrary.org/obo/GOREL_
     - id: ro
+      use_base: TRUE
     - id: pato
-      module_type: minimal
+      use_base: TRUE
+    - id: caro
+      use_base: TRUE
+    - id: CL
+      use_base: TRUE
+    - id: SO
+      make_base: TRUE
+    - id: ncbitaxon
+      mirror_from: http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl
```

Comment:
You should use make_base and also add the correct base iri (be careful of the capitalisation).

Reply:
You mean something like

```
- id: ncbitaxon
  mirror_from: http://purl.obolibrary.org/obo/ncbitaxon/subsets/taxslim.owl
  make_base: TRUE
  base_iris:
    - http://purl.obolibrary.org/obo/NCBITaxon_
```

If so I am not sure I understand why declaring the base IRI would be needed here (I note there is no such declaration in CL and Uberon’ configs).

Reply:
Because robot is case sensitive with base iris and ODK makes an all uppercase assumption

Reply:
Doh! Got it, thanks.